models/questions.py

```diff
@@ -19,6 +21,18 @@
 except:
     pass
 
+db.define_table('question_grades',
```

Comment:
@presnick This needs to be moved to the end of db_ebook.py   Both Jackie and I ran into problems until I figured out that we needed to relocate the definition of this.

I don't have a good explanation for why, but something in the ordering of the model loading was causing big trouble.  Moving it fixes this.